### PR TITLE
fix(community): bump `@upstash/ratelimit` version

### PR DESCRIPTION
**Pull request by CahidArda**
Bumping Upstash Ratelimit version to benefit from the number of improvements we have made.

### CODE REVIEW

**Comment by jacoblee93** [comment]
Flagging that this will break existing users but your call

File: libs/langchain-community/src/callbacks/handlers/upstash_ratelimit.ts [hunk 1]
```diff
@@ -148,7 +148,7 @@ class UpstashRatelimitHandler extends BaseCallbackHandler {
     _name?: string
   ): Promise<void> {
     if (this.tokenRatelimit) {
-      const remaining = await this.tokenRatelimit.getRemaining(this.identifier);
+      const { remaining } = await this.tokenRatelimit.getRemaining(this.identifier);
```
Comment (jacoblee93) [inline libs/langchain-community/src/callbacks/handlers/upstash_ratelimit.ts:151]: May be some way to make this backwards compatible and allow `"^1.1.3 || ^2.0.3"` in deps above

Comment (bracesproul) [reply libs/langchain-community/src/callbacks/handlers/upstash_ratelimit.ts]: @CahidArda I just pushed up a commit allowing those versions

**Review by bracesproul — APPROVED** [review]

**Comment by CahidArda** [comment]
Hi @jacoblee93,

can we go ahead with this PR?

**Comment by jacoblee93** [comment]
Yes, thanks for your patience!

**Pull request merged**